model/clusters_mgmt/v1/manifest_type.model

```diff
@@ -18,4 +18,16 @@ limitations under the License.
 class Manifest {
 	// List of k8s objects to deploy on a hosted cluster.
 	Workloads []Interface
+
+	// Spec of Manifest Work object from open cluster management.
+	Spec Interface
+
+	// Date and time when the manifest got created in clusters-service.
```

Comment:
curious, do we already expose this in the API? Or should we keep something generic like in OCM DB

Reply:
What do you mean? We usually expose for other services, not sure why we wouldn't here